SuperTicTacToe.xcodeproj/project.pbxproj

```diff
@@ -53,6 +53,7 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		1AD3AAD62444B15C00C8651C /* README.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
```

Comment:
I like to keep the README accessible in Xcode (but not part of the build targets).